janrain_capture.module

```diff
@@ -1326,3 +1326,61 @@ function janrain_capture_form_user_profile_form_alter(&$form, &$form_state) {
     }
   }
 }
+function janrain_capture_email_update_batch() {
```

Comment:
These functions are used only during installation / update process. Please move them into `janrain_capture.install` file. hook_update() supports `$sandbox` var. Use it in your functions.